### DISCUSSION TITLE
✅ Powerline Font

### DISCUSSION THREAD

**Discussion by davadev** [isontheline/pro.webssh.net#298] [Ideas]
Any chance to add powerline font support?

**Comment by isontheline** [comment]
Yes tracked issues are :
https://github.com/isontheline/pro.webssh.net/issues/97
https://github.com/isontheline/pro.webssh.net/issues/60

I have a problem to include them into the WebView but it should be possible

**Comment by davadev** [comment]
Ah thank you. Sorry then for duplicate.

**Reply by isontheline** [reply]
No problem for duplicate 😉 your comment will allow me to push up this feature

**Comment by isontheline** [comment]
@davadev Could you check the latest WebSSH Beta Release?
https://testflight.apple.com/join/QSrBK59z

**Comment by davadev** [comment]
Awesome!! Thanks!
